client/src/app/core/repositories/base-repository.ts

```diff
@@ -137,25 +143,50 @@ export abstract class BaseRepository<V extends BaseViewModel, M extends BaseMode
 
     /**
      * Saves the update to an existing model. So called "update"-function
+     * Provides a default procedure, but can be overwritten if required
+     *
      * @param update the update that should be created
      * @param viewModel the view model that the update is based on
      */
-    public abstract async update(update: Partial<M>, viewModel: V): Promise<void>;
+    public async update(update: object, viewModel: BaseViewModel): Promise<void> {
+        const sendUpdate = new this.baseModelCtor();
+        sendUpdate.patchValues(viewModel.getModel());
+        sendUpdate.patchValues(update);
+        return await this.dataSend.partialUpdateModel(sendUpdate);
```

Comment:
Please change this to to a `updateModel`, because the whole model is updated.

Check motion, changeRecommendation and agendaItem updates, becaused they used PATCH (`partialUpdateModel`) instead of PUT. If this does not work, readd the old behaviour in the repos where the update fails.